worker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:18.04
+MAINTAINER Rodrigo Alfonso
+
+# Install repositories
+RUN apt-get update -y
+RUN apt-get install -y software-properties-common
```

Comment:
Compact all those lines in a single line, in order to levarage cache al reduce image size. See for example https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34

Also, remove unused files after installing.